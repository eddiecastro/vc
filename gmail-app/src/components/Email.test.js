import React from "react";
import { render } from "@testing-library/react";
import { Email } from ".";

test("it renders the component", () => {
  const props = {
    id: 1,
    sender: " Test 1",
    subject: "lorum ipsum",
    date: "2020-07-08T07:01:43.511Z",
    tag: "Personal"
  };

  const { container } = render(
    <Email
      id={props.id}
      sender={props.sender}
      subject={props.sender}
      subject={props.subject}
      date={props.date}
      tag={props.tag}
    />
  );

  expect(container.firstChild).toMatchSnapshot();
});
