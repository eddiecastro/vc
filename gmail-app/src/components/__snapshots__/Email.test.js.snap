// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the component 1`] = `
<tr
  class=""
>
  <td
    class=""
  >
    <button
      aria-label="email-checkbox"
      class="ui button"
      style="background-color: transparent;"
    >
      <div
        class="ui fitted checkbox checkbox"
      >
        <input
          class="hidden"
          readonly=""
          tabindex="0"
          type="checkbox"
          value="1"
        />
        <label />
      </div>
    </button>
     Test 1
  </td>
  <td
    class=""
  >
    lorum ipsum
  </td>
  <td
    class=""
  >
    <a
      class="ui blue label"
      style="margin-left: 30px;"
    >
      work
    </a>
    <a
      class="ui green label"
      style="margin-left: 30px;"
    >
      travel
    </a>
  </td>
  <td
    class=""
  >
    2020-07-08T07:01:43.511Z
  </td>
</tr>
`;
